src/visualization/types/Histogram/options.tsx

```diff
@@ -56,14 +56,11 @@ const HistogramOptions: FC<Props> = ({properties, results, update}) => {
   }
 
   const setDomain = (axis: string, domain: [number, number]): void => {
-    let bounds: [string | null, string | null]
+    let bounds: [number | null, number | null]
 
     if (domain) {
       const [min, max] = domain
-      bounds = [
-        min === null ? null : String(min),
-        max === null ? null : String(max),
-      ]
+      bounds = [min === null ? null : min, max === null ? null : max]
```

Comment:
```suggestion
      bounds = [min, max]
```

now that we're not casting to a string, we don't need this anymore - it doesn't actually do anything. Looks like this was put in to prevent `new String(null)` which returns `"null"` rather than `null`

Reply:
Nice...I went ahead and did that for both the Histogram and Heatmap, and also the Scatterplot which is exactly the same function. This does call to attention the rather unsatisfying fact that the same function is duplicated in 3 different places, but at least it's consistent!